targets/targets.json

```diff
@@ -4174,6 +4174,12 @@
         "detect_code": ["4600"],
         "extra_labels_add": ["RTL8195A"]
     },
+    "REALTEK_RTL8710AF": {
+        "inherits": ["MCU_RTL8195A"],
+        "detect_code": ["4601"],
+        "device_has": ["I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "SERIAL", "SPI", "FLASH"],
```

Comment:
@DanielDmlee : From the parent target (due to the inheritance), are all the "device_has" features available in this MCU as well? if yes, maybe it's useful to remove this line / rather use _remove to "remove" the non-available ones?

Reply:
8710AF is the small brother of 8195AM, the function is a limited then 8195AM. Is there usage or sample coed of _remove?

Reply:
@ashok-rao : I updated target.json file after modified to '_remove'